core/vm/analysis_test.go

```diff
@@ -59,12 +59,9 @@ func TestJumpDestAnalysis(t *testing.T) {
 	}
 }
 
-const analysisCodeSize = 1200 * 1024
-
 func BenchmarkJumpdestAnalysis_1200k(bench *testing.B) {
 	// 1.4 ms
-	code := make([]byte, analysisCodeSize)
-	bench.SetBytes(analysisCodeSize)
+	code := make([]byte, 1200000)
```

Comment:
should we still keep analysisCodeSize? it seems readable than 1200000

Reply:
it's low influence code, 1024 * 1024 simualr